.changeset/metal-dogs-fail.md

```diff
@@ -0,0 +1,10 @@
+---
+"@ultraviolet/form": patch
+"@ultraviolet/icons": patch
+"@ultraviolet/illustrations": patch
+"@ultraviolet/plus": patch
+"@ultraviolet/themes": patch
+"@ultraviolet/ui": patch
+---
+
+Update 'contributing.md' to add a necessary command during the installation
```

Comment:
We don't do changeset for non-functional changes (documentation, testing, etc)

Reply:
ok, i did it because i had a message saying that the changeset was missing, i'll remove it

Reply:
No worries, thanks for the contribution!